_posts/en/newsletters/2019-05-21-newsletter.md

```diff
@@ -0,0 +1,221 @@
+---
+title: 'Bitcoin Optech Newsletter #47'
+permalink: /en/newsletters/2019/05/21/
+name: 2019-05-21-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes the bip-anyprevout soft fork proposal, summarizes
+a few technical talks from the Magical Crypto Friends conference, and
+includes our regular sections on bech32 sending support and notable
+changes to popular Bitcoin infrastructure projects.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Proposed anyprevout sighash modes:** two weeks ago, Anthony Towns
+  posted a proposed BIP to the Bitcoin-Dev and Lightning-Dev mailing
+  lists for consideration.  The idea, [bip-anyprevout][], provides the
+  functionality previously described by [BIP118][] with modifications to
+  work with Taproot and to reduce the risk of accidental misuse.  One of
+  the signature hashing (sighash) modes available with bip-anyprevout is
```

Comment:
I think this jumps into talking about 'sigature hashing (sighash) modes' a little too quickly, without introducing the concept first. Perhaps just one additional introductory sentence would help:

' ... and to reduce the risk of accidental misuse.  The BIP proposes two news ways for a signature to commit to the coin that it spends. The first of these signature hashing (sighash) modes is directly compatible ...'